metalsmith/index.js

```diff
@@ -1,20 +1,32 @@
 const metalsmith = require('metalsmith');
 const layouts = require('metalsmith-layouts');
+const inPlace = require('metalsmith-in-place');
+const registerHelpers = require('metalsmith-register-helpers');
 
 metalsmith(__dirname)
-.source('../src') // Source directory
-.destination('../build') // Destination directory
-.clean(true) // Clean destination before
-.ignore(['app', 'assets', 'layouts', 'partials']) // Ignore other files than handlebars template
-.use(layouts({ // Wrap layouts around html
-	engine: 'handlebars', // Use the layout engine you like
-	rename: true,
-	directory: '../src/layouts',
+.source('../src')
+.destination('../build')
+.clean(true) // Clean destination before build
+.ignore(['app', 'assets', 'handlebars']) // Ignore other files than handlebars template
+.use(registerHelpers({
+	directory: '../src/handlebars/helpers'
+}))
+.use(layouts({ // Wrap layouts around content pages
+	engine: 'handlebars',
+	rename: false,
+	directory: '../src/handlebars/layouts',
 	default: 'default.hbs',
-	pattern: '**/*.hbs',
-	partials: '../src/partials',
+	pattern: '*.hbs',
+	partials: '../src/handlebars/partials',
 	partialExtension: '.hbs'
-})).build((error) => { // Build process
+}))
+.use(inPlace({ // Render handlebars content pages
```

Comment:
`metalsmith-in-place` is needed in addition to `metalsmith-layouts` to render handlebars (partials, variables from frontmatters and helpers).

Without `inPlace` handlebars stuff is only rendered in layout files.
I have the following two testfiles to to reproduce and check this:

```
src/handlebars/layouts/blank.hbs
src/test.hbs
```

See the discussion to see why this is needed:
https://github.com/superwolff/metalsmith-layouts/issues/128